src/mgr/OSDPerfMetricTypes.h

```diff
@@ -6,6 +6,7 @@
 
 #include "include/denc.h"
 #include "include/stringify.h"
+#include "common/Formatter.h"
```

Comment:
Why do we need to include Formatter here?

Reply:
moving it to .cc file